src/DynamoCoreWpf/Views/Core/CustomNodeAnnotationView.xaml

```diff
@@ -0,0 +1,323 @@
+﻿<UserControl Name="Group"
```

Comment:
Do we need a completely new xaml here or can we refactor the group view into components that can be used between two different annotation type views?